.github/workflows/build.yml

```diff
@@ -50,7 +50,6 @@ jobs:
 
             - name: Run analysis
               run: composer analyse
-              if: matrix.symfony-version != '6.0.*'
```

Comment:
Tell me please why do we run static analysis on different versions??

Reply:
Different versions of dependencies have different APIs - eg. what works with Symfony 4 might not work with Symfony 5.